modules/oxide/src/less/theme/components/comments/comment.less

```diff
@@ -307,6 +307,10 @@
     overflow: auto;
     padding: 12px;
     scroll-behavior: smooth;
+
+    &:has(textarea:focus-within) {
```

Comment:
I wonder if `:focus` would be enough here since focus-within focus at or on a child within and the textarea has no children.